src/js/components/modals/ServiceScaleFormModal.js

```diff
@@ -106,6 +106,17 @@ class ServiceScaleFormModal extends mixin(StoreMixin) {
           id: service.id,
           scaleBy: parseInt(instances, 10)
         });
+      } else if (service instanceof Pod) {
+        let spec = Object.assign({}, service.getSpec().get(),
```

Comment:
Renamed to `ServiceSpecUtil` and removed the `Pods` namespace inside it, as we discussed.